http/oidc/src/main/java/org/wildfly/security/http/oidc/TokenValidator.java

```diff
@@ -76,20 +76,28 @@ private TokenValidator(Builder builder) {
         this.clientConfiguration = builder.clientConfiguration;
     }
 
+    public VerifiedTokens parseAndVerifyToken(final String idToken, final String accessToken,
+                                              OidcHttpFacade.Cookie cookie) throws OidcException {
```

Comment:
Just realized, I guess the cookie is something that is specifically used for the nonce validation case, right? So perhaps there's some more cleanup that would be good to do so this method would just need to take the idToken and accessToken. 

Looking back at the original PR, `RefreshableOidcSecurityContext` was updated so that it was possible to pass it the session random value cookie but that's no longer needed now.

Reply:
fixed